packages/material/scss/chip/_theme.scss

```diff
@@ -0,0 +1,49 @@
+@import "~@progress/kendo-theme-default/scss/chip/_theme.scss";
+
+@include exports( "chip/theme/material" ) {
+    .k-selection-single {
+        .k-chip-selected {
+            &.k-chip-outline {
```

Comment:
reduce the level of nesting as putting k-chip-selected and k-chip-outline on the same line selector